components/dashboard/StepsCard.tsx

```diff
@@ -39,13 +41,16 @@ export const StepsCard: React.FC<StepsCardProps> = ({
     <DelayedDisplay delay={delay}>
       <Card
         flexDir="column"
-        py={6}
-        px={6}
+        p={8}
         gap={4}
         position="relative"
         cursor={isComplete ? (isOpen ? "default" : "pointer") : undefined}
         onClick={isComplete ? () => !isOpen && setIsOpen(true) : undefined}
-        bg="linear-gradient(158.84deg, rgba(255, 255, 255, 0.05) 13.95%, rgba(255, 255, 255, 0) 38.68%)"
+        bg={
```

Comment:
@assimovt You can actually use useColorModeValue, to get the value. Just nicer syntax.

```
  const bg = useColorModeValue('red.500', 'red.200')
  // Then use the value returned, in this example red.500 is light mode and red.200 is dark mode
```